### PR TITLE
Change labels to indicate AAM available to pro

### PR DESCRIPTION
**Pull request by SeBBBe**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-330.surge.sh

Updated pages:
- [Advanced metrics](https://neo4j-docs-aura-330.surge.sh/aura/auradb/managing-databases/advanced-metrics)
- [Advanced metrics](https://neo4j-docs-aura-330.surge.sh/aura/aurads/managing-instances/advanced-metrics)

File: modules/ROOT/pages/aurads/managing-instances/advanced-metrics.adoc [hunk 1]
```diff
@@ -1,8 +1,6 @@
 [[aura-monitoring]]
 = Advanced metrics
 
-label:AuraDS-Enterprise[]
```
Comment (SeBBBe) [inline modules/ROOT/pages/aurads/managing-instances/advanced-metrics.adoc:4]: As we don't have a free tier of AuraDS, all AuraDS users will have access to AAM, so removing the label.

File: modules/ROOT/pages/auradb/managing-databases/advanced-metrics.adoc [hunk 2]
```diff
@@ -1,6 +1,7 @@
 [[aura-monitoring]]
 = Advanced metrics
 
+label:AuraDB-Professional[]
 label:AuraDB-Enterprise[]
```
Comment (SeBBBe) [inline modules/ROOT/pages/auradb/managing-databases/advanced-metrics.adoc:5]: Will be available for AuraDB Pro and Enterprise (but not free!). Certain features limited to Enterprise, as indicated by labels in the following text.

**Review by fiquick — APPROVED** [review]
Looks good

**Pull request merged**